mmv1/products/monitoring/MonitoredProject.yaml

```diff
@@ -31,6 +31,7 @@ description: "A [project being monitored](https://cloud.google.com/monitoring/se
 immutable: true
 error_retry_predicates: ['transport_tpg.IsMonitoringPermissionError']
 custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_import: templates/terraform/custom_import/monitoring_monitored_project.go.erb
```

Comment:
Why do we need a custom import here? Is it only to support importing with the `v1/` prefix?

Reply:
It's there to shorten the `name` and `metrics_scope` fields in the import id.